src/main/java/org/folio/qm/controller/RecordsEditorRecordsApi.java

```diff
@@ -26,6 +29,11 @@ public RecordsEditorRecordsApi(MarcRecordsService marcRecordsService) {
     return ResponseEntity.ok(quickMarc);
   }
 
+  @Override
+  public ResponseEntity<Void> recordsPost(@Valid QuickMarc quickMarc) {
```

Comment:
Is this method supposed to return Void but not the newly created data?

Reply:
for the current story- yes